assets/test_github.yaml

```diff
@@ -9,6 +9,7 @@ dataset:
     reads:
       - reads: /home/runner/work/genomeassembly/genomeassembly/Undibacterium_unclassified/genomic_data/baUndUnlc1/hic-arima2/41741_2#7.sub.cram
 hic_motif: GATC,GANTC,CTNAG,TTAA
+hic_aligner: minimap2
```

Comment:
My understanding is that the default aligner should be "bwamem2", while "minimap2" is to be used predominantly for big genomes. I'd suggest to keep this CI test configuration as well as test.yaml with the primary HiC mapping implementation i.e. "bwamem2" and add a test particularly focused on testing functionality for big genomes.

Reply:
should I switch it back to bwamem2 for CI test for now then.

Reply:
I would have done that! :)